documentation/src/docs/asciidoc/user-guide/running-tests.adoc

```diff
@@ -205,7 +200,38 @@ on the dependency-aggregating JUnit Jupiter artifact similar to the following.
 [subs=attributes+]
 ----
 dependencies {
-	testImplementation("org.junit.jupiter:junit-jupiter:{jupiter-version}") // version can be omitted when using the BOM
+    testImplementation("org.junit.jupiter:junit-jupiter:{jupiter-version}")
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
+}
+----
+
+Alternatively, you can use Gradle's
+https://docs.gradle.org/current/userguide/jvm_test_suite_plugin.html[JVM Test Suite]
```

Comment:
Is that feature "ready for prime time"?

Reply:
Officially it's still incubating but it's been there for so long that I don't think they'll remove it.